### ISSUE TITLE
Replicatation of RoBERTA Base in Table 3 

### ISSUE THREAD

**Issue by gaopengcuhk** [pytorch/fairseq#1111]
I want to replicate table 3 performance in your paper.

Batch size 8K

 srun -p clusterRTX --gres gpu:8 -c 24 --pty fairseq-train --fp16 ./data-bin/wiki-book/ --task masked_lm --criterion masked_lm     --arch roberta_base --sample-break-mode complete --tokens-per-sample 512    0.9, 0.98)' --adam-eps 1e-6 --clip-norm 0.0     --lr-scheduler polynomial_decay --lr 0.0007 --warmup-updates 10000 --total-num-update 125000     --dropout 0.1 --attention-dropout 0.1 --weight-decay 0.01     --max-sentences 16 --update-freq 64 --max-updaog-interval 1 --save-dir checkpoints_roberta --ddp-backend=no_c10d

When I evaluate the SST-2 performance, I can not reach the claimed performance listed in your paper 92.8.
RoBERTA Epoch   1       |      2     |       3    |    4     |       5     |      6      |        7      |        8      |        9       
 SST-2 fintune    80.45  |   81.15 |     83.4  |   83.4 |     86.93|    89.24  |      90.45 |       90.90|      92.04  

 10     |    11    |      12
91.81 |  91.59  |  

Each epoch has 1176 iterations. Which means for epoch 11, the iteration is 12936 which is only 1/3 of 31000 steps listed in your paper. However, there appear over-fitting is happening.

**Comment by gaopengcuhk** [comment]
additional information
currently, ppl is 5.32 for epoch 11

**Comment by gaopengcuhk** [comment]
additional information

12           |      13(ppl 4.75)
92.04      |      92.04

**Comment by huihuifan** [comment]
can you sweep a few parameters like the batchsize and the learning rate for SST2? In the different RoBERTa settings, slightly different parameters may be optimal for finetuning

**Comment by myleott** [comment]
Batch size should not be 8K for SST-2. Remove `--update-freq` and you'll get the expected results :)
